config/config.go

```diff
@@ -156,6 +156,11 @@ var (
 		RefreshInterval: model.Duration(5 * time.Minute),
 	}
 
+	// DefaultECSSDConfig is the default ECS SD configuration.
+	DefaultECSSDConfig = ECSSDConfig{
+		RefreshInterval: model.Duration(60 * time.Second),
```

Comment:
I don't see how having a shared filesystem hurts, but of any ecosystem ECS is likely one of the largest so I can see how it can make sense to have it natively.